appinfo/info.xml

```diff
@@ -5,76 +5,64 @@
     <name>External Storage: Swarm</name>
     <summary>Bringing Swarm to Nextcloud - decentralized file-storage, connected to the blockchain</summary>
     <description><![CDATA[
-# Nextcloud Swarm Plugin
+# **Hejbit - Nextcloud Swarm Plugin**
 
-*Decentralized Sovereign cloud-storage comes to Nextcloud!*
+*Bring decentralized, sovereign cloud storage to Nextcloud with the Hejbit Swarm plugin!*
 
-This plugin brings [Swarm](https://www.ethswarm.org/) to [Nextcloud](https://nextcloud.com) - decentralized file-storage, connected to the blockchain.
+This plugin integrates [Swarm](https://www.ethswarm.org/) (a decentralized, blockchain-connected file storage system) directly into your [Nextcloud ](https://nextcloud.com/)instance.
 
-## Table of Contents
+## **Table of Contents**
 
-- [Dependencies](#dependencies)
-- [Setup and Installation](#setup-and-installation)
-- [Usage](#usage)
-	- [Settings](#settings)
-	- [Using Swarm](#using-swarm)
-- [License](#license)
+- [Dependencies](https://apps.nextcloud.com/apps/files_external_ethswarm#dependencies) .
```

Comment:
@frozenbanana The point after the link was only a fix to prevent opening the thumbnail on collectives. You can remove it from here.